types/tizen-common-web/tslint.json

```diff
@@ -3,6 +3,7 @@
     "rules":{
         "max-line-length": false,
         "no-empty-interface": false,
-        "interface-over-type-literal": false
+        "interface-over-type-literal": false,
+        "no-misused-new": false        
```

Comment:
We cannot accept disabling rules in new changes. Please fix error or disable it on the line it needs to instead so it doesnt disable it everywhere and can be reviewed for the cases where it needs to turn it off. Also disabling this rule is never a good sign so please leave a comment pointing to where you disable for easy review. Thanks